lib/common/http1client.c

```diff
@@ -380,6 +382,25 @@ static void on_connection_ready(struct st_h2o_http1client_private_t *client)
     h2o_timeout_link(client->super.ctx->loop, client->super.ctx->io_timeout, &client->_timeout);
 }
 
+SSL_SESSION *fetch_ssl_session_cache(h2o_http1client_t *client)
+{
+    h2o_cache_ref_t *ref = h2o_cache_fetch(client->ctx->ssl_session_cache, client->ctx->loop->_now, client->ssl.session_cache_key, client->ssl.session_cache_key_hash);
```

Comment:
`h2o_now` should be called instead of using `_now` field directly. `h2o_loop_t` when using `libuv` does not have `_now` field.